src/create_update.c

```diff
@@ -227,6 +227,25 @@ static int check_build_env(void)
 	return 0;
 }
 
+static int check_group_file(void)
+{
+	int ret = -1;
+
+	// Ensure the os-core group is defined
+	while (1) {
```

Comment:
Just from a stylistic perspective, a while (ret) would be preferable to a while (1) with a break;